imports/wasi_snapshot_preview1/testdata/zig/wasi.zig

```diff
@@ -19,9 +19,10 @@ pub fn main() !void {
                 return;
             },
         };
-        var iter = dir.iterate();
-        while (try iter.next()) |entry| {
-            try stdout.print("./{s}\n", .{entry.name});
+
+        try ls(dir);
```

Comment:
note: something here resulted in the wasm being 215KB larger than before. that's 2700%